awestruct.gemspec

```diff
@@ -37,7 +37,6 @@ Haml and Markdown filters are touchy things. Redcarpet or Rdiscount work well if
   s.add_dependency 'tilt', '~> 2.0', '>= 2.0.1'
   s.add_dependency 'mime-types', '~> 2.1'
   s.add_dependency 'rest-client', '~> 1.7', '>= 1.7.2'
-  s.add_dependency 'ruby-s3cmd', '~> 0.1', '>= 0.1.5'
```

Comment:
Yeah, if the gem isn't enough then yes, put that info in there

On Wednesday, April 22, 2015, Guillaume Grossetie <notifications@github.com>
wrote:

> In awestruct.gemspec
> <https://github.com/awestruct/awestruct/pull/345#discussion_r28851200>:
>
> > @@ -37,7 +37,6 @@ Haml and Markdown filters are touchy things. Redcarpet or Rdiscount work well if
> >    s.add_dependency 'tilt', '~> 2.0', '>= 2.0.1'
> >    s.add_dependency 'mime-types', '~> 2.1'
> >    s.add_dependency 'rest-client', '~> 1.7', '>= 1.7.2'
> > -  s.add_dependency 'ruby-s3cmd', '~> 0.1', '>= 0.1.5'
>
> I think it would be better to update the documentation to mention that
> s3cmd must be available in the PATH. What do you think ?
>
> I don't think I can implement similar behavior using the gem because
> advanced commands are hard (impossible ?) to write using this abstraction.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/awestruct/awestruct/pull/345/files#r28851200>.
>


-- 
Jason Porter
http://en.gravatar.com/lightguardjp